CoreFoundation/Base.subproj/CoreFoundation_Prefix.h

```diff
@@ -307,29 +304,21 @@ CF_EXPORT int _NS_access(const char *name, int amode);
 #define WIN32_LEAN_AND_MEAN
 
 #ifndef WINVER
-#define WINVER  0x0501
+#define WINVER  0x0601
```

Comment:
This is basically defining the min deployment target for Windows, correct?

Reply:
Not really, this is more like the SDK version, which automatically implies the "deployment target" is not higher than that.

You could set WINVER to 0x0603 (Windows 8.1) for example, and then use LoadLibrary/GetProcAddress (dlopen/dlsym), to conditionally use APIs introduced in Windows 8.1, so that your app still works when running on Windows 7 (0x0601). If you have WINVER at 0x0601, then the definitions of Windows 8.1 APIs would not be visible to you in the SDK headers.

0x0601 seems like a good value here (means: "allows statically binding to APIs introduced no later than Windows 7"), I can't imagine anyone is interested in deploying Swift code to anything less than Windows 7 at this point in time.